buildpipeline/DotNet-CoreFx-Trusted-Mac-Native.json

```diff
@@ -343,11 +343,11 @@
   "defaultBranch": "refs/heads/master",
   "queue": {
     "pool": {
-      "id": 39,
-      "name": "DotNet-Build"
+      "id": 83,
```

Comment:
Don't update OSX, there are no OSX machines in the testing pool and no plan to update the agents at this time.